### DISCUSSION TITLE
How to Delete Messages from a Channel Using a Bot

### DISCUSSION THREAD

**Discussion by cauamp** [python-telegram-bot/python-telegram-bot#4221] [Q&A]
Essentially, I have a bot in a channel, and I want to create a function that, when called, simply deletes all the messages in the channel specified by the CHAT_ID. Is it possible?

**Comment by Poolitzer** [comment]
Sure, if you keep track of all message ids

**Reply by MrAntsLife** [reply]
Oh. Active not only on Telegram but also on GitHub. I found you 😂😂😂
